### PR TITLE
Fix staking txn replication issue

### PR DESCRIPTION
**Pull request by rlan35**
Tested locally.

### CODE REVIEW

File: api/proto/node/node.go [hunk 1]
```diff
@@ -137,6 +139,21 @@ func ConstructTransactionListMessageAccount(transactions types.Transactions) []b
 	return byteBuffer.Bytes()
 }
 
+// ConstructStakingTransactionListMessageAccount constructs serialized staking transactions in account model
+func ConstructStakingTransactionListMessageAccount(transactions types2.StakingTransactions) []byte {
+	byteBuffer := bytes.NewBuffer([]byte{byte(proto.Node)})
```
Comment (fxfactorial) [inline api/proto/node/node.go]: maybe `[]byte{byte(proto.Node), byte(Staking), byte(Send)})`?

Comment (rlan35) [reply api/proto/node/node.go]: Ack. Let's do it later.

File: node/node.go [hunk 2]
```diff
@@ -256,6 +257,21 @@ func (node *Node) tryBroadcast(tx *types.Transaction) {
 	}
 }
 
+func (node *Node) tryBroadcastStaking(stakingTx *staking.StakingTransaction) {
+	msg := proto_node.ConstructStakingTransactionListMessageAccount(staking.StakingTransactions{stakingTx})
+
+	shardGroupID := nodeconfig.NewGroupIDByShardID(nodeconfig.ShardID(0)) // broadcast to beacon chain
+	utils.Logger().Info().Str("shardGroupID", string(shardGroupID)).Msg("tryBroadcastStaking")
+
+	for attempt := 0; attempt < NumTryBroadCast; attempt++ {
```
Comment (LeoHChen) [inline node/node.go]: will it causing problem to resend the same staking tx multiple times in retry?  will it create multiple validators on the same account?

**Comment by LeoHChen** [comment]
also, please create a new branch on top of the master and squash your commits.  The current PR will cause messy commit history.

File: node/node.go (hunk 2 above)
Comment (rlan35) [reply node/node.go]: No, the p2p network will screen that out if it's the same message. Also transactions can't be repeated because of nonce.

**Pull request closed without merging**